lib/api/CT-list-update-service.js

```diff
@@ -49,15 +50,18 @@ CTListUpdateService.prototype._updateCTList = async function() {
                     content.crowdtangle_list_account_pairs[account.id] = [list.metadata.title];
                 } 
                 else {
-                    content.crowdtangle_list_account_pairs[account.id].push(list.metadata.title);
+                    content.crowdtangle_list_account_pairs[account.id].indexOf(list.metadata.title) === -1 && content.crowdtangle_list_account_pairs[account.id].push(list.metadata.title); //if listname is not in the array, then it will be added. 
```

Comment:
This line got complex! Can we please use `else if (condition) { stuff }` instead of `else { condition && stuff }`